README.md

```diff
@@ -51,33 +51,33 @@ The Format is:
 
 Where `<base-encoding-character>` is used according to the multibase table.
 
-### Multibase Table v1.0.0-RC (semver)
+### Multibase Table
 
 The current multibase table is [here](multibase.csv):
 
 ```
-encoding,           code, description
-identity,           0x00, 8-bit binary (encoder and decoder keeps data unmodified)
-base2,              0,    binary (01010101)
-base8,              7,    octal
-base10,             9,    decimal
-base16,             f,    hexadecimal
-base16upper,        F,    hexadecimal
-base32hex,          v,    rfc4648 no padding - highest char
-base32hexupper,     V,    rfc4648 no padding - highest char
-base32hexpad,       t,    rfc4648 with padding
-base32hexpadupper,  T,    rfc4648 with padding
-base32,             b,    rfc4648 no padding
-base32upper,        B,    rfc4648 no padding
-base32pad,          c,    rfc4648 with padding
-base32padupper,     C,    rfc4648 with padding
-base32z,            h,    z-base-32 (used by Tahoe-LAFS)
-base58flickr,       Z,    base58 flicker
-base58btc,          z,    base58 bitcoin
-base64,             m,    rfc4648 no padding
-base64pad,          M,    rfc4648 with padding - MIME encoding
-base64url,          u,    rfc4648 no padding
-base64urlpad,       U,    rfc4648 with padding
+encoding,          code, description,                                              status
+identity,          0x00, 8-bit binary (encoder and decoder keeps data unmodified), recommendation
```

Comment:
recommendation is kind of weird term. What about `default` or `base set`?

Reply:
I'm using https://github.com/libp2p/specs/blob/master/00-framework-01-spec-lifecycle.md (based on https://www.w3.org/2004/02/Process-20040205/tr.html#maturity-levels?).

But "draft", "candidate", and "default" is definitely clearer for this use-case.